mindsdb/interfaces/knowledge_base/controller.py

```diff
@@ -126,6 +128,15 @@ def insert(self, df: pd.DataFrame):
         df_emb = self._df_to_embeddings(df)
         df = pd.concat([df, df_emb], axis=1)
 
+        # drop original 'content' column if it exists
+        if TableField.CONTENT.value in df.columns:
+            df = df.drop(TableField.CONTENT.value, axis='columns')
```

Comment:
The original content we can name something else.. I dropped it because we have to rename the `embedding_context` as the content to preserve the rest of the functionality.